portality/migrate/20190828_regenindex_dataset_change/README.md

```diff
@@ -0,0 +1,8 @@
+*S.E. 2019-08-28*
+
+Save (and re-generate index) for Application (Suggestion) and Journals
+Refresh the journal information stored within each Article, save (and re-generate index) each Article
+
+Run
+
+    python portality/upgrade.py -u portality/migrate/20190828_regenindex_dataset_change/migrate.json
```

Comment:
Can we also include the issue number in the migration directory name, like `20190828_1234_regen..`